massa-graph/src/bootstrapable_graph.rs

```diff
@@ -22,35 +22,23 @@ use std::ops::Bound::Included;
 /// Bootstrap graph
 #[derive(Debug, Clone, Serialize, Deserialize)]
 pub struct BootstrapableGraph {
-    /// Map of active blocks, were blocks are in their exported version.
-    pub active_blocks: Map<BlockId, ExportActiveBlock>,
-    /// Best parents hashes in each thread.
-    pub best_parents: Vec<(BlockId, u64)>,
-    /// Latest final period and block hash in each thread.
-    pub latest_final_blocks_periods: Vec<(BlockId, u64)>,
-    /// Head of the incompatibility graph.
-    pub gi_head: Map<BlockId, Set<BlockId>>,
-    /// List of maximal cliques of compatible blocks.
-    pub max_cliques: Vec<Clique>,
+    /// list of final blocks
+    pub final_blocks: Vec<ExportActiveBlock>,
```

Comment:
Can you explain why we don't need the rest ?

Reply:
basically we won't be transferring non-final blocks (so the cliques we send are empty)